rules/library.bzl

```
@@ -579,6 +579,7 @@ def apple_library(
     testonly = kwargs.pop("testonly", False)
     features = kwargs.pop("features", [])
     plugins = kwargs.pop("plugins", None)
+    swift_always_include_developer_search_paths = kwargs.pop("swift_always_include_developer_search_paths", None)
```

I agree this should be 
```suggestion
    swift_always_include_developer_search_paths = kwargs.pop("always_include_developer_search_paths", None)
```

we didnt make that distinction for `plugins`

---

See [this](https://github.com/bazel-ios/rules_ios/pull/852#issuecomment-2018890660)

---

Essentially a user would have to look at the code here to understand they should use `swift_always_include_developer_search_paths` vs. the actual flag `always_include_developer_search_paths`

---

Ok, people will try to use flags by the original names. I'm convinced, cc @mattrobmattrob haha

---

https://github.com/bazel-ios/rules_ios/pull/852/commits/4a57fd23f7b82dfb8d26c33599dacc866a6f6c0e